advisories/CriticalReactorFailure.svg

```diff
@@ -8,7 +8,7 @@
    version="1.1"
```

Comment:
Changes to other SVGs rename functionally-unused Inkscape properties, for completeness.